src/pages/ProductDetail/ProductDetail.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useCallback, useState } from 'react'
+import { useParams, useNavigate } from 'react-router-dom';
+import Header from '../../components/Header/Header';
+import styles from'./ProductDetail.module.css';
+import ProductInfo from './components/ProductInfo/ProductInfo';
+import { getProductDetail } from '../../api/itemApi';
+import Comment from './components/Comment/Comment';
+import backicon from '../../assets/images/backicon.png';
+
+function ProductDetail() {
+  const { productId } = useParams();
+  const [item, setItem] = useState(null);
+  const [loading, setLoading] = useState(true);
+  let navigate = useNavigate();
```

Comment:
P1:
재할당을 해야하거나 꼭 필요한 경우가 아니라면 const를 사용해주세요~

```suggestion
  const navigate = useNavigate();
```